modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/RotateGestureRecognizer.java

```diff
@@ -25,76 +25,72 @@
 
 package com.sun.javafx.tk.quantum;
 
-import com.sun.glass.events.KeyEvent;
-import com.sun.glass.events.TouchEvent;
-
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Set;
-import javafx.util.Duration;
-import javafx.event.EventType;
-import javafx.scene.input.RotateEvent;
-import javafx.animation.Animation;
+import java.util.concurrent.TimeUnit;
+
+import com.sun.glass.events.KeyEvent;
+import com.sun.glass.events.TouchEvent;
+
 import javafx.animation.Interpolator;
 import javafx.animation.KeyFrame;
 import javafx.animation.KeyValue;
 import javafx.animation.Timeline;
 import javafx.beans.property.DoubleProperty;
 import javafx.beans.property.SimpleDoubleProperty;
+import javafx.scene.input.RotateEvent;
+import javafx.util.Duration;
 
 class RotateGestureRecognizer implements GestureRecognizer {
-    private ViewScene scene;
+    private static final double MAX_INITIAL_VELOCITY = 500;
+    private static final double ROTATION_INERTIA_MILLIS = 1500;
+    private static final long ROTATION_INERTIA_THRESHOLD_NANOS = TimeUnit.MILLISECONDS.toNanos(300);
+
+    // gesture will be activated if |rotation| > rotationThresholdDegrees
+    private static double rotationThresholdDegrees = 5;
+    private static boolean rotationInertiaEnabled = true;
 
-    // gesture will be activated if |rotation| > ROTATATION_THRESHOLD
-    private static double ROTATATION_THRESHOLD = 5; //in degrees
-    private static boolean ROTATION_INERTIA_ENABLED = true;
-    private static double MAX_INITIAL_VELOCITY = 500;
-    private static double ROTATION_INERTIA_MILLIS = 1500;
     static {
         @SuppressWarnings("removal")
         var dummy = AccessController.doPrivileged((PrivilegedAction<Void>) () -> {
             String s = System.getProperty("com.sun.javafx.gestures.rotate.threshold");
             if (s != null) {
-                ROTATATION_THRESHOLD = Double.valueOf(s);
+                rotationThresholdDegrees = Double.valueOf(s);
```

Comment:
should we catch a NumberFormatException and re-throw it with a meaningful text message?  Or the current behavior is ok?

ditto on line 64